packages/collector/test/test_util/ProcessControls.js

```diff
@@ -211,6 +219,10 @@ class ProcessControls {
     }
   }
 
+  getProcessLogs() {
```

Comment:
This is a real cool feature for our ProcessControls. You can now easily ask for the logs of the forked process by setting `WITH_FULL_STDIO`.